docs/source/simulators/SimulationPathFramework.rst

```diff
@@ -49,9 +49,9 @@ Per default, the backend uses the :code:`sequential` strategy. For this particul
  - then, the first CNOT is applied (:code:`[4, 2] -> 5`)
  - finally, the last CNOT is applied (:code:`[5, 3] -> 6`)
 
- The corresponding simulation path is thus described by :code:`[[0, 1], [4, 2], [5, 3]]` and the final state is the one with ID :code:`6`.
+ The corresponding simulation path is thus described by :code:``[[0, 1], [4, 2], [5, 3]]`` and the final state is the one with ID :code:`6`.
```

Comment:
Not quite sure the double backticks are intentional here. RST tags work with single backticks.